src/components/bills/Progress.svelte

```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
+	import VoteCard from '$components/VoteCard/VoteCard.svelte';
+	import { BillStatus, type Bill } from '$models/bill';
+	import { EventStatus, type Event, EventActionType } from '$models/event';
+	import {
+		ArrowRight,
+		CheckmarkFilled,
+		CircleDash,
+		DocumentMultiple_02,
+		Misuse
+	} from 'carbon-icons-svelte';
+	import RoyalGazette from './RoyalGazette.svelte';
+	import { Button } from 'carbon-components-svelte';
+	import BillCard from '$components/BillCard/BillCard.svelte';
+	import type { VoteCardProps } from '../../routes/assemblies/[id]/+page';
+	import type { Voting } from '$models/voting';
+	import type { VotingResultSummary } from '../../routes/bills/[id]/+page';
```

Comment:
I think we need to refactor this. Component should be independent from the route so we should not import anything from the route.

1. Please check if the `VoteCardProps` is the same with`VoteCard` component props. Then we can use 
```ts
import type { ComponentProps } from 'svelte';

ComponentProps<VoteCard>
```
2. We can declare the type definition and export it in `<script lang="ts" context="module">` see `DataPage` component for an example. Then the route should import from here.